server/services/educationAndWorkPlanService.ts

```diff
@@ -29,6 +30,20 @@ export default class EducationAndWorkPlanService {
     }
   }
 
+  async getGoalsByStatus(prisonNumber: string, status: GoalStatusValue, token: string): Promise<Goals> {
+    try {
+      const response = await this.educationAndWorkPlanClient.getGoalsByStatus(prisonNumber, status, token)
+      return { goals: toGoals(response), problemRetrievingData: false }
+    } catch (error) {
+      if (error.status === 404) {
+        logger.debug(`No plan created yet so no goals with [${status}] for Prisoner [${prisonNumber}]`)
```

Comment:
Ah ha, this answers my questions above re: whether a 404 is right and whether a 404 would trigger the "the service is down" message. Got it 👍